### PR TITLE
LPD-23367: Hide Detect Category Names from CSV File checkbox in Data Migration Center

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
- Hide `Detect Category Names from CSV File` check box because it is not being used. However, it is not deleted because this could be useful for any moment.

**Note:** No need to create a test case for this because we are not changing the behavior of this checkbox which was disabled all the time.

**Before PR**
Screenshot from 2024-04-19 11-25-26

**After PR**
Screenshot from 2024-04-19 11-25-52

**JIRA Related Ticket**
https://liferay.atlassian.net/browse/LPD-23367

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp [hunk 1]
```diff
@@ -96,7 +96,7 @@ renderResponse.setTitle(editable ? LanguageUtil.get(request, "edit-template") :
 								/>
 							</clay:alert>
 
-							<div class="mt-2">
+							<div class="mt-2" hidden>
```
Comment (jaimelr10) [inline modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp:99]: What do you think of `<div class="mt-2 d-none">`?

**Review by jaimelr10 — COMMENTED** [review]
I think both work, what do you think of this?

**Comment by sergiojimcos** [comment]
ci:forward

File: modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp (hunk 1 above)
Comment (sergiojimcos) [reply modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp]: Much better!!

**Pull request closed without merging**